specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -211,6 +228,213 @@ union EvictionPolicy {
   Deallocate: "Deallocate",
 }
 
+@added(Versions.v2024_11_01)
+@doc("AdditionalCapabilities for VM.")
+model AdditionalCapabilities {
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "ultraSSDEnabled which does not follow camel casing"
+  @doc("The flag that enables or disables a capability to have one or more managed data disks with UltraSSD_LRS storage account type on the VM or VMSS. Managed disks with storage account type UltraSSD_LRS can be added to a virtual machine or virtual machine scale set only if this property is enabled.")
```

Comment:
Add Line breaks in doc.
Use this format: @doc("""
   The flag that enables or disables a capability to have one or more managed data disks with UltraSSD_LRS storage account type on the VM or VMSS.
 Managed disks with storage account type UltraSSD_LRS can be added to a virtual machine or virtual machine scale set only if this property is enabled.
    """)

Reply:
updated, as suggested.